plexupdate.sh

```diff
@@ -481,33 +493,33 @@ if [ "${SKIP_DOWNLOAD}" == "no" ]; then
 	echo "OK"
 fi
 
-if [ ! "${PLEXSERVER}" = "" -a "${AUTOINSTALL}" == "yes" ]; then
+if [ ! -z "${PLEXSERVER}" -a "${AUTOINSTALL}" = "yes" ]; then
 	# Check if server is in-use before continuing (thanks @AltonV, @hakong and @sufr3ak)...
 	if ! wget --no-check-certificate -q -O - https://${PLEXSERVER}:32400/status/sessions | grep -q '<MediaContainer size="0">' ; then
 		echo "Server ${PLEXSERVER} is currently being used by one or more users, skipping installation. Please run again later"
 		cronexit 6
 	fi
 fi
 
-if [ "${AUTOINSTALL}" == "yes" ]; then
+if [ "${AUTOINSTALL}" = "yes" ]; then
 	sudo ${DISTRO_INSTALL} "${DOWNLOADDIR}/${FILENAME}"
 fi
 
-if [ "${AUTODELETE}" == "yes" ]; then
-	if [ "${AUTOINSTALL}" == "yes" ]; then
+if [ "${AUTODELETE}" = "yes" ]; then
+	if [ "${AUTOINSTALL}" = "yes" ]; then
 		rm -rf "${DOWNLOADDIR}/${FILENAME}"
 		echo "Deleted \"${FILENAME}\""
 	else
 		echo "Will not auto delete without [-a] auto install"
 	fi
 fi
 
-if [ "${AUTOSTART}" == "yes" ]; then
-	if [ "${REDHAT}" == "no" ]; then
+if [ "${AUTOSTART}" = "yes" ]; then
+	if [ "${REDHAT}" = "no" ]; then
 		echo "The AUTOSTART [-s] option may not be needed on your distribution."
 	fi
 	# Check for systemd
-	if [ -f "/bin/systemctl" ]; then
+	if ! hash systemctl 2>/dev/null; then
 		systemctl start plexmediaserver.service
 	else
 		/sbin/service plexmediaserver start
```

Comment:
Should use hash here as well and give a warning if we don't know how to auto-start the service.